website/docs/api/plugins/plugin-content-blog.mdx

```diff
@@ -131,6 +132,14 @@ type CreateFeedItemsFn = (params: {
 }) => Promise<BlogFeedItem[]>;
 ```
 
+#### `ProcessBlogPostsFn` {#ProcessBlogPostsFn}
+
+```ts
+type ProcessBlogPostsFn = (params: {
+  blogPosts: BlogPost[];
+}) => void | BlogPost[];
```

Comment:
```suggestion
}) => Promise<void | BlogPost[]>;
```

Async support probably didn't make much sense for sorting, but for more general processing it might be useful